src/Components/ArtworkGrid/__tests__/ArtworkGrid.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { mount } from "enzyme"
+import { cloneDeep } from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { renderRelayTree } from "../../../DevTools"
+import { ArtworkGridItem } from "../../Artwork/GridItem"
+import { ArtworkGridFixture } from "../__stories__/ArtworkGridFixture"
```

Comment:
If following the co-located pattern above, would it make sense to put this fixture along side it's component as well? 

One thing to note is that over in the `__tests__/Apps/Fixtures` folder is we've been starting to mirror the overall folder structure as well so that fixture data is easier to find at a glance. (Some of it needs to be refactored.)